# the task is a python script
bool has_script
string script

# the task is a sequence of predefined functions 
string function
string[] args
string return_var